---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
    collapse = TRUE,
    comment = "#>",
    fig.path = "man/figures/README-",
    out.width = "100%"
)
```

# mlBioNets

<!-- badges: start -->

[![GitHub issues](https://img.shields.io/github/issues/Nertekkad/mlBioNets)](https://github.com/Nertekkad/mlBioNets/issues) [![GitHub pulls](https://img.shields.io/github/issues-pr/Nertekkad/mlBioNets)](https://github.com/Nertekkad/mlBioNets/pulls)
[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
[![check-bioc](https://github.com/Nertekkad/mlBioNets/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/Nertekkad/mlBioNets/actions/workflows/check-bioc.yml)
<!-- badges: end -->

**\*\*mlBionets\*\*** is an R package designed to comprehensively analyze microbiota data using a multilayer network approach. This package enables researchers to integrate and analyze multiple types of biological data, providing a robust framework for studying complex microbial communities and their interactions within different environmental and biological contexts.

**Features**

-   **Multilayer Network Construction:** Build and visualize multilayer networks from microbiota data, incorporating various layers such as phylogenetic, functional, and environmental information.

-   **Data Integration:** Seamlessly integrate different types of microbiota data, including taxonomic profiles and metadata

-   **Advanced Analysis Tools:** Perform network-based analyses, including community detection, network centrality, and inter-layer connectivity.

-   **Visualization:** Generate publication-quality visualizations to illustrate complex network structures and interactions.

-   **Customizable Workflows:** Easily customize analysis workflows to suit specific research needs, facilitating reproducible and flexible data analysis.

## Installation instructions

You can install the development version of mlBioNets like so:

```{r 'Intallation', eval = requireNamespace('mlBioNets'), results='hide'}
devtools::install_github("Nertekkad/mlBioNets")
```

## Citation

Below is the citation output from using `citation('mlBioNets')` in R. Please run this yourself to check for any updates on how to cite **mlBioNets**.

```{r 'citation', eval = requireNamespace('mlBioNets')}
print(citation('mlBioNets'), bibtex = TRUE)
```

Please note that the `mlBioNets` was only made possible thanks to many other R and bioinformatics software authors, which are cited either in the vignettes and/or the paper(s) describing this package.

## Code of Conduct

Please note that the `mlBioNets` project is released with a [Contributor Code of Conduct](http://bioconductor.org/about/code-of-conduct/). By contributing to this project, you agree to abide by its terms.
